### PR TITLE
Backport 7.56.0 changelog

### PR DESCRIPTION
**Pull request by Pythyu**
### What does this PR do?

Backport changelog of the 7.56.0 release

### CODE REVIEW

**Review by davidor — APPROVED** [review]

File: CHANGELOG-DCA.rst [hunk 1]
```diff
@@ -2,6 +2,51 @@
...
+
+7.56.0
+======
+
+.. _Release Notes_7.56.0_Prelude:
+
+Prelude
+-------
+
+Released on: 2024-08-16
```
Comment (jennchenn) [inline CHANGELOG-DCA.rst:15]: just curious: what does this date correspond to?

Comment (Pythyu) [reply CHANGELOG-DCA.rst:15]: When the tag was made I believe but since we had a few issues for this release the gap is noticeable indeed

Comment (jennchenn) [reply CHANGELOG-DCA.rst:15]: Ah makes sense, thanks!

**Review by jennchenn — APPROVED** [review]

**Review by AliDatadog — APPROVED** [review]

**Pull request merged**